pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/VedtakEndringAvAlderspensjonInstitusjonsopphold.kt

```diff
@@ -0,0 +1,444 @@
+                    }
+                }
+
+                showIf(saksbehandlerValg.informasjonOmSivilstandVedInstitusjonsopphold) {
+                    // infoBrukerInst_001
+                    paragraph {
+                        textExpr(
+                            Bokmal to "Fordi du ikke bor sammen med ".expr() + fritekst("ektefelle/partner/samboer") + " mens du er på institusjon får du pensjon som enslig pensjonist. Det vil si at pensjonen din først øker til " + fritekst("beløp") + " kroner før skatt fra og med " + pesysData.krav.virkDatoFom.format() + ", men deretter blir " + fritekst("redusert/stanset") + " fra " + fritekst("dato fom") + ".",
+                            Nynorsk to "Fordi du ikkje bur saman med ".expr() + fritekst("ektefelle/partnar/sambuar") + " mens du er på institusjon, får du pensjon som einsleg pensjonist. Det vil si at pensjonen din først aukar til " + fritekst("beløp") + " kroner før skatt frå og med " + pesysData.krav.virkDatoFom.format() + ", men deretter blir " + fritekst("redusert/stansa") + " frå " + fritekst("dato fom") + ".",
+                            English to "Because you are not living with your ".expr() + fritekst("spouse/partner/cohabitant") + "  when you are institutionalized, your pension will be calculated as if you are a single pensioner. This means that first your pension payments will increase to NOK " + fritekst("beløp") + " before tax from " + pesysData.krav.virkDatoFom.format() + ", but then will be " + fritekst("reduced/suspended") + " from " + fritekst("dato fom") + ".",
+                        )
+                    }
+                }
+            }
+
+            showIf(pesysData.beregnetPensjonPerManedVedVirk.totalPensjon.greaterThan(1)) {
+                // utbetalingsInfoMndUtbet_001
+                paragraph {
+                    text(
+                        Bokmal to "Hvis du har andre pensjonsytelser som for eksempel AFP eller tjenestepensjon, blir de utbetalt i tillegg til alderspensjonen. Alderspensjonen din utbetales innen den 20. hver måned. Du finner oversikt over utbetalingene dine på $UTBETALINGER_URL.",
+                        Nynorsk to "Dersom du har andre pensjonsytingar som for eksempel AFP eller tenestepensjon, kjem slik utbetaling i tillegg til alderspensjonen. Alderspensjonen din blir betalt ut innan den 20. i kvar månad. Du finn meir informasjon om utbetalingane dine på $UTBETALINGER_URL.",
+                        English to "If you have occupational pensions from other schemes, this will be paid in addition to your retirement pension. Your pension will be paid at the latest on the 20th of each month. See the more detailed information on what you will receive at $UTBETALINGER_URL."
+                    )
+                }
+            }
+
+
+            // pre-selected, 2
+            showIf(pesysData.alderspensjonVedVirk.regelverkType.isOneOf(AP2011, AP1967)) {
+                // hjemmelBrukerAP2011Inst_001
+                paragraph {
```

Comment:
Virker så rart for meg at vi ikke gir noe hint til hva som kanskje skal slettes. Det som er default fra før av er at det som bare dukker opp uten noe valg får de ikke lov å slette. Kan ikke dette skape masse feil.